mantle/util/retry.go

```diff
@@ -59,7 +59,11 @@ func WaitUntilReady(timeout, delay time.Duration, checkFunction func() (bool, er
 
 		time.Sleep(delay)
 
+		// In the past we have seen WaitUntilReady take much longer than
+		// timeout duration to run. The logging below will help gather more info
```

Comment:
optional.. we're slightly changing the definition of `timeout duration` so maybe make this more futureproof:

```suggestion
		// Log how long it took WaitUntilReady to run. This will help gather information about
		// how long it takes remote API requests (like provisioning machines) to finish.
```